src/main/java/uk/gov/hmcts/sptribs/recordlisting/LocationService.java

```diff
@@ -38,8 +39,11 @@ public class LocationService {
     private LocationClient locationClient;
 
     public DynamicList getHearingVenuesByRegion(String regionId) {
-        final var hearingVenues = getCourtVenues(regionId);
-        return populateVenueDynamicList(hearingVenues);
+        final HearingVenue[] hearingVenues = getCourtVenues(regionId);
+        HearingVenue[] filteredHearingVenus = hearingVenues == null ? null : Arrays.stream(hearingVenues)
+                .filter(v -> COURT_TYPE_ID.equals(v.getCourtTypeId())).toArray(HearingVenue[]::new);
```

Comment:
Typo: filteredHearingVenues

What advantage is there to an array HearingVenue[], here?
Why not use a List<HearingVenue>?